composer.json

```diff
@@ -14,8 +14,8 @@
 		"nikic/fast-route": "0.6.*",
 		"psr/log": "~1.0",
 		"stack/builder": "~1.0",
-		"symfony/http-foundation": ">=2.4",
-		"symfony/http-kernel": ">=2.4"
+		"symfony/http-foundation": ">=3.4",
```

Comment:
Why not 4.x

Reply:
3.4 is a LTS, so as long as it passes my tests, I'll allow it :)